DemoServer/Controllers/Demos/TextSearch/FTSQuerySearchRanking/metadata.json

```diff
@@ -0,0 +1,121 @@
+{
+    "Slug": "fts-query-search-ranking",
+    "SourceFileName": "FTSQuerySearchRankingController.cs",
+    "Title": "FTS - Query - Search Ranking",
+    "DescriptionLines": [
+        "**Full Text Search query** allows searching for documents containing particular term(s) within a block of text.",
+        "",
+        "**The order of the resulting documents** depends on the searched terms order.",
```

Comment:
"depends on the searched terms order"
did you mean "operator" here?
asking since literally reading this sentence gives the impression that "term1 term2" will produce different ordering than "term2 term1"

Reply:
Ah, this is true indeed, but only for OR operator? We could clarify this a bit

Reply:
1. Now I'm thinking we should change this demo name from `Ranking` to just `Basic`
as it is actually a basic search query...  

2. Will mention in the steps that you can search on a single term (as this demo searches on 2 terms)

3. And, I think I'd better remove this unclear line from the description - as it is also explained in the relevant Step